tests/manage/z_cluster/upgrade/test_check_mds_cache_memory_limit_post_upgrade.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    post_ocs_upgrade,
+    ManageTest,
+    skipif_ocs_version,
+    bugzilla,
+)
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+log = logging.getLogger(__name__)
+
+
+@post_ocs_upgrade
+@skipif_ocs_version("<4.7")
+@bugzilla("1951348")
+@bugzilla("1944148")
+@pytest.mark.polarion_id("OCS-2554")
+class TestToCheckMDSCacheMemoryLimit(ManageTest):
+    """
+    Validate post ocs upgrade mds cache memory limit
+
+    """
+
+    def test_check_mds_cache_memory_limit(self):
```

Comment:
Since we are checking the ceph defaults can this moved under: tests/manage/z_cluster/test_ceph_default_values_check.py ?

Reply:
done